src/nvim/terminal.c

```diff
@@ -217,10 +214,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
+    vterm_state_set_bold_highbright(state, true);
```

Comment:
I don't think it has to be there. It is just there because #3333 had it there.

I didn't move it down because I didn't know if you wanted to change the check to be something else later.

I can move it down now if you want to.